webpack.config.babel.js

```diff
@@ -20,7 +20,7 @@ const config = {
 	module: {
 		rules: [
 			{
-				test: /\.jsx?$/,
+				test: /\.js?$/,
```

Comment:
lately i have been only using `.jsx` on files with `import React ...` at or any JSX in the file and `.js` in the rest. I'm fine with using `.js` on all files.